Money*/Money*/Scenes/Dashboard/DashboardViewController.swift

```diff
@@ -10,15 +10,54 @@ import UIKit
 import Firebase
 
 final class DashboardViewController: UIViewController {
+    // MARK: - IBOutlets
+    @IBOutlet private var walletCollectionView: UICollectionView!
+    @IBOutlet private var walletPageControl: UIPageControl!
+    @IBOutlet private var addTransactionButton: UIButton!
+    @IBOutlet private var reportButton: UIButton!
+    @IBOutlet private var budgetButton: UIButton!
+    @IBOutlet private var showAllTransactionsButton: UIButton!
+    @IBOutlet private var transactionTableView: UITableView!
+    
     // MARK: - Private properties
     private var user: User!
     
     // MARK: - Private functions
     override func viewDidLoad() {
         super.viewDidLoad()
+        configureSubviews()
+        registerCustomCells()
         addAuthorizationListener()
     }
     
+    private func configureSubviews() {
+        addTransactionButton.makeRoundedAndShadowed()
+        showAllTransactionsButton.makeRounded()
+        reportButton.makeRoundedAndShadowed(cornerRadius: reportButton.frame.height / 2)
+        budgetButton.makeRoundedAndShadowed(cornerRadius: budgetButton.frame.height / 2)
+        
+        [reportButton, budgetButton].forEach {
+            $0?.imageEdgeInsets = UIEdgeInsets(top: 10, left: 10, bottom: 10, right: 10)
+            
+        }
+        
+        walletCollectionView.collectionViewLayout = CardFlowLayout()
```

Comment:
```suggestion
        walletCollectionView.collectionViewLayout = CardFlowLayout().then {
                $0.dataSource = self
                $0.delegate = self
                $0.numberOfPages = 5
                $0.currentPage = 0
        }
```
dùng lib Then cho gọn anh nhé